builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,28 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
+		local check_path = gamedetails.path .. DIR_DELIM .."menu" ..
+					DIR_DELIM .. identifier .. "." .. i .. ".png"
+		local file = io.open(check_path,"rb")
+		if file ~= nil then
+			count = i
+			io.close(file)
+			break
+		end
+	end
+	local new_path = gamedetails.path .. DIR_DELIM .."menu" .. DIR_DELIM ..
+		identifier .. "." .. math.random(1, count) .. ".png"
+	local exists = io.open(new_path,"rb")
```

Comment:
Okay, I'll try and do that when I get back. I'm not entirely sure how I would accomplish it though...